migrate-base.js

```diff
@@ -91,6 +85,12 @@ async function migrateBase({ web3, spinner, confirm, opts, logTx, previousMigrat
 		GenesisProtocol
 	);
 
+	// deploy Reputation
+	await deploy(
```

Comment:
why do we need to deploy Reputation as part of the base contracts?

Reply:
agree . it should be part of the dao creation ..created automatically using the daoCreator.

Reply:
Because the native reputation created by the daoCreator is owned by the ucontroller and we need a reputation we can use for the testing in the subgraph.

Reply:
@ben-kaufman 
Then, you somehow need to connect this reputation to the dao created in `migrate-dao`.
But I think there are better ways. For example, consider registering some account as a scheme with full permissions on the DAO, so he can mint/burn reputation.

Reply:
Sounds like a good idea, will try implementing that